scripts/generate-usage-data/generate-usage-data.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime, timedelta, timezone
+from pprint import pprint
+import random
+import uuid
+import json
+
+import requests
+
+token = '<API Token>'
+EnterpriseCustomer = 'customer_id'
+StandardCustomer = 'customer_id'
```

Comment:
these should come in from the command line args, check out the argparser, it also supports a list of args, its super neat!

Reply:
I prefer it this way, but would be open to understand what you think command line args are better